docs/blog/newsletter-november-2023.rst

```diff
@@ -0,0 +1,123 @@
+#########################################
+
+Salutations, documentarians. I hope you are able to find things in your life to be thankful for and also the time to be able to appreciate them. I'm thankful for the wonderful community of documentarians at Write the Docs and the insights they freely share with us every month.
+
+Have you filled out the `2023 Documentation Salary Survey <https://salary-survey.writethedocs.org/>`_ yet? If you have, we appreciate you - is there a friend or colleague you could spread the word to? If you haven't, please consider adding your data - the insights become more valuable with every submission. 
+
+A special call to contractors, freelancers and self-employed people: to publish meaningful median rates, we still need a lot more data. The questions for non-employees have been updated to cater to a variety of independent work situations, and if they don't fit for you, please let us know in the feedback section of your submission how we can make it better.
+
+In conference news, the Australia conference has now published its `full schedule </conf/australia/2023/news/announcing-speakers/>`__. Be sure to `book a ticket </conf/australia/2023/tickets/>`__ while you still can. And the Portland conference has announced `dates for 2024 </conf/portland/2024/news/welcome/>`__, so you can start making plans already.
+
+The newsletter this month covers whether and how to publish logs of changes to docs (not products), how the organization of docs teams might affect people on them, and how to measure the quality of docs.
+
+-------------------
+Changelogs for Docs
+-------------------
+
+Technical writers often publish product-focused changelogs or release notes that describe new product features and fixes but may also publish changelogs that list new and updated content in the docs themselves. Tech writers might need to provide a docs changelog to satisfy standards like ISO, inform technical support engineers of new docs that can help with support cases, or ensure that stakeholders like marketing and product know about docs they can refer to in their materials. Docs changelogs are also useful for highlighting the quality and volume of the work the docs team does, building goodwill and enhancing the team's reputation throughout the organization.
+
+Some writers collect their own commit messages as the basis for docs changelogs, either manually or with a tool based on a standard like `Conventional Commits <https://www.conventionalcommits.org/en/v1.0.0/>`_ that helps automate changelog builds. The `TinyMCE changelog <https://github.com/tinymce/tinymce-docs/blob/staging/docs-6/changelog.md>`__ is an example of a commit-based changelog. Others rely on features that are built in to the documentation tools they already use, like page histories that make it easier to figure out what changed. When you're deciding what to include, focus on new content and substantial updates and omit typo corrections and style changes.
```

Comment:
End of this paragraph reads like full opinion.

Reply:
I was thinking about that too but second guessed myself.
```suggestion
Some writers collect their own commit messages as the basis for docs changelogs, either manually or with a tool based on a standard like `Conventional Commits <https://www.conventionalcommits.org/en/v1.0.0/>`_ that helps automate changelog builds. The `TinyMCE changelog <https://github.com/tinymce/tinymce-docs/blob/staging/docs-6/changelog.md>`__ is an example of a commit-based changelog. Others rely on features that are built in to the documentation tools they already use, like page histories that make it easier to figure out what changed. When you're deciding what to include, consider focusing on new content and substantial updates and perhaps omitting typo corrections and style changes.
```